easybuild/tools/testing.py

```diff
@@ -295,11 +295,20 @@ def post_pr_test_report(pr_nrs, repo_type, test_report, msg, init_session_state,
     if system_info['cpu_arch_name'] != UNKNOWN:
         system_info['cpu_model'] += " (%s)" % system_info['cpu_arch_name']
 
+    # add GPU info, if known
+    gpu_info = get_gpu_info()
+    gpu_str = ""
+    if gpu_info:
+        for vendor in gpu_info:
+            for gpu, num in gpu_info[vendor].items():
+                gpu_str += ", %s %s x %s" % (vendor, num, gpu)
```

Comment:
```suggestion
                gpu_str += ", %s x %s %s" % (num, vendor, gpu)
```

Personally I prefer `1 x NVIDIA Tesla V100-SXM2-16GB` because I think it is easier to read compared to `NVIDIA 1 x Tesla V100-SXM2-16GB`.